### PR TITLE
Nv mss containing henok

### PR DESCRIPTION
**Pull request by nafisa-valieva**
Hello! I have introduced tiny-tiny changes to the manuscripts containing Henok to make them appear in the ´red box'. I have limited my intervention to LIT1340EnochE.

### CODE REVIEW

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, that is very useful! Could you please add a change element describing your addition to each of the files, so that your work will be acknowledged?

**Comment by nafisa-valieva** [comment]
> Thank you, that is very useful! Could you please add a change element describing your addition to each of the files, so that your work will be acknowledged?

should I?

**Comment by thea-m** [comment]
I think so, what would would speak against it? It is the official workflow, and the only way to make your work become easily visible on the website. You can also acknowledge potential other sources in the change element, e.g. "added ID LIT1340EnochE based on article XX"

File: EMML/6001-7000/EMML6686.xml [hunk 1]
```diff
@@ -71,8 +83,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                </physDesc>
                <history>
                   <origin>
-                     <origDate notBefore="1600" notAfter="1700">17th century,
-                                                        </origDate>
+                     <origDate notBefore="1600" notAfter="1700">17th century, </origDate>
```
Comment (karljonaskarlsson) [inline EMML/6001-7000/EMML6686.xml:86]: Maybe even just "17th century"?

File: EMML/7001-8000/EMML7103.xml [hunk 2]
```diff
@@ -65,9 +70,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                </physDesc>
                <history>
                   <origin>
-                     <origDate notBefore="1659" notAfter="1660">17th century,
-                                                        1659/60</origDate>
-                     <note>Date given as Year of the World 7152 on f. 27a: ተፈጸመ፡ ዝንቱ፡ መጸሐፈ፡ ሄኖክ፡ በ፸፻ወ፩፻፶ወ፪ዓመት፡ እምፍጥረተ፡ ዓለም፡ በዘመነ፡ የሐንስ፨</note>
+                     <origDate notBefore="1659" notAfter="1660">17th century, 1659/60</origDate>
+                     <note>Date given as Year of the World 7152 on f. 27a: ተፈጸመ፡ ዝንቱ፡ መጸሐፈ፡ ሄኖክ፡
```
Comment (karljonaskarlsson) [inline EMML/7001-8000/EMML7103.xml:74]: Maybe "መ<sic resp="NV">ጸ</sic>ሐፈ፡"? (I checked the ms)

File: EMML/7001-8000/EMML7103.xml [hunk 3]
```diff
@@ -65,9 +70,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
...
+                        በ፸፻ወ፩፻፶ወ፪ዓመት፡ እምፍጥረተ፡ ዓለም፡ በዘመነ፡ የሐንስ፨</note>
```
Comment (karljonaskarlsson) [inline EMML/7001-8000/EMML7103.xml:75]: ዮሐንስ (I checked the ms)

File: EMML/7001-8000/EMML7584.xml [hunk 4]
```diff
@@ -9,38 +10,53 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
...
                   </altIdentifier>
-                  
+
                </msIdentifier>
-               <msContents><summary>ff. 1r–27v: 1 Enoch —ff. 28r–40r: Job110; ff. 41r–42v: Song of Songs; ff. 43r–48v: Tobit; ff. 49r–61v: Proverbs; ff. 61v–66r: Ecclesiastes; ff. 66r–73v: Wisdom of Solomon; ff. 74r–81r: Esther; ff. 81r–89v: Judith; ff. 90r–115v: Isaiah; ff. 116r–127r: Daniel; ff. 128r–158v: Jeremiah; ff. 158v–160r: Baruch; ff. 160r–162r: Lamentations; ff. 162rv: Epistle of Jeremiah; f. 163r: Prophecy of Jeremiah to Pashur; ff. 163r–167v: 4 Baruch; ff. 168r–194v: Ezekiel; ff. 195r–199r: Hosea; ff. 199r–202v: Amos; ff. 202v–205r: Micah; ff. 205r–206v: Joel; f. 207r: Obadiah; ff. 207v–208v: Jonah; ff. 208v–209v: Nahum; ff. 209v–210v: Habakkuk; ff. 210v–212r: Zephaniah; ff. 212r–213r: Haggai; ff. 213r–218r: Zechariah; ff. 218rv: Malachi; ff. 219r–230v: 1 Ezra; ff. 230v–239v: 2 Ezra; ff. 240r–260v: Sirach; ff. 261r–279r: 1 Maccabees; ff. 279v–284v: 2 Maccabees111 (incomplete)
-               </summary></msContents>
+               <msContents>
+                  <summary>ff. 1r–27v: 1 Enoch —ff. 28r–40r: Job110; ff. 41r–42v: Song of Songs; ff.
```
Comment (karljonaskarlsson) [inline EMML/7001-8000/EMML7584.xml:43]: Do we know what "Job110" means?

File: EMML/7001-8000/EMML7584.xml [hunk 5]
```diff
@@ -9,38 +10,53 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
...
+                     Wisdom of Solomon; ff. 74r–81r: Esther; ff. 81r–89v: Judith; ff. 90r–115v:
+                     Isaiah; ff. 116r–127r: Daniel; ff. 128r–158v: Jeremiah; ff. 158v–160r: Baruch;
+                     ff. 160r–162r: Lamentations; ff. 162rv: Epistle of Jeremiah; f. 163r: Prophecy
+                     of Jeremiah to Pashur; ff. 163r–167v: 4 Baruch; ff. 168r–194v: Ezekiel; ff.
+                     195r–199r: Hosea; ff. 199r–202v: Amos; ff. 202v–205r: Micah; ff. 205r–206v:
+                     Joel; f. 207r: Obadiah; ff. 207v–208v: Jonah; ff. 208v–209v: Nahum; ff.
+                     209v–210v: Habakkuk; ff. 210v–212r: Zephaniah; ff. 212r–213r: Haggai; ff.
+                     213r–218r: Zechariah; ff. 218rv: Malachi; ff. 219r–230v: 1 Ezra; ff. 230v–239v:
+                     2 Ezra; ff. 240r–260v: Sirach; ff. 261r–279r: 1 Maccabees; ff. 279v–284v: 2
+                     Maccabees111 (incomplete) </summary>
```
Comment (karljonaskarlsson) [inline EMML/7001-8000/EMML7584.xml:54]: Same, what does "Maccabees111" mean? Is it a copypasted page number?

**Review by karljonaskarlsson — APPROVED** [review]

**Comment by thea-m** [comment]
@karljonaskarlsson , I think that the errors you rightly highlighted were already present in the files to which Nafisa only added one ID, and editing the whole files was not part of her work process right now. That is not to say that they may not be corrected, if anyone is in the mood, but this was not the main focus :)

**Comment by karljonaskarlsson** [comment]
> @karljonaskarlsson , I think that the errors you rightly highlighted were already present in the files to which Nafisa only added one ID, and editing the whole files was not part of her work process right now. That is not to say that they may not be corrected, if anyone is in the mood, but this was not the main focus :)

Yes, no corrections are obligatory :) I just noted what struck me in the parts that had been changed.

**Review by thea-m — APPROVED** [review]

**Comment by nafisa-valieva** [comment]
> > @karljonaskarlsson , I think that the errors you rightly highlighted were already present in the files to which Nafisa only added one ID, and editing the whole files was not part of her work process right now. That is not to say that they may not be corrected, if anyone is in the mood, but this was not the main focus :)
> 
> Yes, no corrections are obligatory :) I just noted what struck me in the parts that had been changed.
> 
> And some things, like these numbers within the text identifications, may even be difficult to correct, without figuring out what is going on… but on the other hand, this could be an occasion to correct them. If someone is in the mood :D

I will merge the changes and then the files will be available for further improvements (by someone else, not me)!

File: EMIP/1-1000/EMIP00937.xml [hunk 6]
```diff
@@ -30,16 +31,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                  <summary>Hosea (1r-6v), Amos (6v-10v), Micah (10v-14r), Joel (14r-16r), Obadiah (16rv), Jonah (16v-17v), Nahum (17v-19r), Habakkuk (19r-20v), Zephaniah (20v-22r), Haggai (22r-23r), Zechariah (23r-29r), Malachi (29r-31r), ? (31rv), Jeremiah (32r-84v), Henoch with marginal commentary (85r-120r), ? (120r)</summary>
+                  <summary>Hosea (1r-6v), Amos (6v-10v), Micah (10v-14r), Joel (14r-16r), Obadiah
+                     (16rv), Jonah (16v-17v), Nahum (17v-19r), Habakkuk (19r-20v), Zephaniah
+                     (20v-22r), Haggai (22r-23r), Zechariah (23r-29r), Malachi (29r-31r), ? (31rv),
+                     Jeremiah (32r-84v), Henoch with marginal commentary (85r-120r), ?
+                     (120r)</summary>
                   <msItem xml:id="ms_i1">
-                     <textLang mainLang="gez">Gǝ‘ǝz</textLang>
+                     <locus from="85r" to="120r"/>
+                     <title ref="LIT1340EnochE">Henok</title>
```
Comment (eu-genia) [inline EMIP/1-1000/EMIP00937.xml:41]: I have not seen this one.

This cannot be ms_i1. 

i1 are Prophets
then you have i1.1 Hosea, i1.2 Amos etc.

or no ms_i at all

Thank you

File: EMML/6001-7000/EMML6686.xml [hunk 7]
```diff
@@ -9,40 +10,51 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
...
+                     Songs; ff. 70r–95v: Isaiah; ff. 96r–99v: Hosea; ff. 99v–103r: Amos; ff.
+                     103r–105v: Micah; ff. 105v–107r: Joel; ff. 107rv: Obadiah; ff. 107v–108v:
+                     Jonah; ff. 108v–109v: Nahum; ff. 109v–110v: Habakkuk; ff. 110v–111v: Zephaniah;
+                     ff. 112rv: Haggai; ff. 112v–117v: Zechariah; ff. 117v–119r: Malachi; ff.
+                     119r–129v: Daniel105; ff. 130r–149r: Sirach; ff. 149v–178v: Jeremiah; ff.
+                     178v–180r: Baruch; ff. 180r–182r: Lamentations; ff. 182rv: Epistle of Jeremiah;
+                     f. 182v: Prophecy of Jeremiah to Pashur; ff. 182v–186v: 4 Baruch; ff.
+                     187r–214r: Ezekiel </summary>
+                  <msItem xml:id="ms_i1">
+                     <locus from="4r" to="27r"/>
```
Comment (eu-genia) [inline EMML/6001-7000/EMML6686.xml:55]: as above

Comment (eu-genia) [reply EMML/6001-7000/EMML6686.xml:55]: and then someone wants to have all witnesses for Jeremiah and has to redo?

**Pull request merged**